src/BloomExe/Spreadsheet/SpreadsheetExporter.cs

```diff
@@ -82,5 +100,89 @@ private void AddTranslationGroupRow(XmlNode group, string pageNumber, int groupI
 				row.SetCell(index,content);
 			}
 		}
+
+		private void AddXmatterDataRow(XmlNode node, string imagesFolderPath)
+		{
+			var dataBookNodeList = node.SafeSelectNodes("./div[@data-book]").Cast<XmlElement>().ToList();
+			dataBookNodeList.Sort((a, b) => a.GetAttribute("data-book").CompareTo(b.GetAttribute("data-book")));
+			string prevDataBookLabel = null;
+			SpreadsheetRow row = _spreadsheet.Header;
+			foreach (XmlElement dataBookElement in dataBookNodeList)
+			{
+				var dataBookLabel = dataBookElement.GetAttribute("data-book");
+
+				//The first time we see this tag:
+				if (!dataBookLabel.Equals(prevDataBookLabel))
+				{
+					row = new ContentRow(_spreadsheet);
+					row.SetCell(InternalSpreadsheet.MetadataKeyLabel, dataBookLabel.Trim());
+
+					var imageSrcAttribute = dataBookElement.GetAttribute("src").Trim();
+
+					if (IsImageXmatterElement(dataBookElement, dataBookLabel))
+					{
+						if (imageSrcAttribute.Length > 0
+							&& dataBookElement.InnerText.Trim().Length > 0
+							&& !imageSrcAttribute.Equals(dataBookElement.InnerText.Trim()))
+						{
+							var msg = LocalizationManager.GetString("Spreadsheet:XmatterConflictWarning",
+								"Export warning: Found differing 'src' attribute and element text for xmatter element " + dataBookLabel
+								+ ". The 'src' attribute will be ignored.");
+							NonFatalProblem.Report(ModalIf.All, PassiveIf.None, msg, showSendReport: true);
+						}
+
+						string imageSource;
+						if (FirstChildIsImageWithSource(dataBookElement))
+						{
+							if (! dataBookElement.Name.Contains("branding"))
+							{
+								var msg = LocalizationManager.GetString("Spreadsheet:XmatterNonBrandingImageElment",
+									"Export warning: Found a non-branding image in an <img> element for " + dataBookLabel
+									+ ". This is not fully handled yet.");
+								NonFatalProblem.Report(ModalIf.All, PassiveIf.None, msg, showSendReport: true);
+							}
+							imageSource = dataBookElement.ChildNodes[0].GetStringAttribute("src");
+						}
+						else
+						{
+							imageSource = dataBookElement.InnerText.Trim();
+						}
+						row.SetCell(InternalSpreadsheet.ImageSourceLabel, ImagePath(imagesFolderPath, imageSource));
+
+						continue; 
+					}
+				}
+
+				if (IsImageXmatterElement(dataBookElement, dataBookLabel))
+				{
+					var msg = LocalizationManager.GetString("Spreadsheet:XmatterImageMultiple",
+						"Export warning: Found multiple elements for image element " + dataBookLabel + ". Only the first will be exported.");
```

Comment:
Should I do something different in this case? Will there be multiple branding image elements with the same name? Right now the importer won't import multiple images for the same element. I guess I would have to make multiple rows with the same "metadata key" but different content?